nmqtt.nim

```diff
@@ -1103,6 +1105,10 @@ proc set_host*(ctx: MqttCtx, host: string, port: int=1883, sslOn=false) =
   ctx.port = Port(port)
   ctx.sslOn = sslOn
 
+proc set_ssl_certificates*(ctx: MqttCtx, sslCertFile: string, sslKeyFile: string) =
```

Comment:
Can you add code comments like your comment in the Readme? Just append:
```
  ## Sets the SSL Certificate and Key files to use Mutual TLS authentication
```

Reply:
Done